shub/image/deploy.py

```diff
@@ -79,51 +79,41 @@ def deploy_cmd(target, version, username, password, email,
         target_conf.project_id, version, image_name, endpoint, apikey,
         username, password, email)
 
-    utils.debug_log('Deploy with params: {}'.format(params))
+    click.echo("Deploying {}".format(image_name))
+    utils.debug_log('Deploy parameters: {}'.format(params))
     req = requests.post(
         urljoin(endpoint, '/api/releases/deploy.json'),
         data=params,
         auth=(apikey, ''),
         timeout=300,
         allow_redirects=False
     )
-    try:
-        req.raise_for_status()
-    except requests.exceptions.HTTPError:
-        _handle_deploy_errors(req)
-
-    click.echo("Deploy task results: {}".format(req))
+    if req.status_code == 400:
```

Comment:
To my best knowledge Dash catches 401 from releaser and wraps it in the exception that produces a 400 error.